nilearn/surface/surface.py

```diff
@@ -685,17 +686,18 @@ def load_surf_mesh(surf_mesh):
     surf_mesh : str or numpy.ndarray
         Either a file containing surface mesh geometry (valid formats
         are .gii .gii.gz or Freesurfer specific files such as .orig, .pial,
-        .sphere, .white, .inflated) or a list or tuple of two Numpy arrays,
-        the first containing the x-y-z coordinates of the mesh
-        vertices, the second containing the indices (into coords)
-        of the mesh faces.
+        .sphere, .white, .inflated) or two Numpy arrays organized in a list,
+        tuple or a namedtuple with the fields "coordinates" and "faces"
 
     Returns
     --------
-    [coords, faces] : List of two numpy.ndarray
-        The first containing the x-y-z coordinates of the mesh vertices,
-        the second containing the indices (into coords) of the mesh faces.
+    mesh : namedtuple
+        With the fields "coordinates" and "faces", each containing a
+        numpy.ndarray
     """
+    # Create a named tuple
+    Mesh = namedtuple("mesh", ["coordinates", "faces"])
```

Comment:
I think that I would do this definition at the module level, and reuse it as much as possible.

Reply:
Also, do we want the names to be coordinates & faces? In the document, I believe that we had said "points" and "triangles", but I am not to hung up on those names. What matters to me is that the names make as many people happy as possible.

Reply:
I'm +1 on "coordinates" and "faces" personally, but that's likely since I'm more familiar with the neuroimaging literature, which uses those terms !

Reply:
I am happy with either, I used coordinates and faces because that was what we used in the previous function, but since the arguments there were positional and not actually named it also wouldnt be a big issue to change

Reply:
OK, so I am updating the proposal to use "coordinates" and "faces"

Reply:
> I think that I would do this definition at the module level, and reuse it as much as possible.

Meaning in surface.py or in surface/\_\_init\_\_.py  ?